unit-tests/src/test/java/org/eclipse/collections/impl/map/mutable/primitive/AbstractMutableObjectBooleanMapTestCase.java

```diff
@@ -97,6 +97,17 @@ public void getOrThrow()
         Assert.assertFalse(map1.getOrThrow(null));
     }
 
+    @Test
+    public void getAndPut()
+    {
+        MutableObjectBooleanMap<Integer> map1 = this.getEmptyMap();
+        Assert.assertTrue(map1.getAndPut(1, false, true));
```

Comment:
Please explicitly box the keys here so it is clear.

Reply:
fixed